src/web_interface/templates/database/database_advanced_search.html

```diff
@@ -112,42 +125,6 @@ <h4>Structure of file objects in database:</h4>
 - tags                  # User defined tags, e.g. Partial, Crawled
 - vendor                # e.g. EvilCorp
 - version               # e.g 1.0.0.312</pre>
-
-            <hr>
-
-            <h4>Structure of processed_analysis field:</h4>
-
-            {% if database_structure is mapping %}
```

Comment:
Is the `database_structure` variable used anywhere else? If it isn't is there maybe some python code that can be cleaned up too?

Reply:
You are absolutely right. Removed unused `database_structure` from code and more.

Reply:
Also removed varietyjs installation and config entries. I hope I didn't miss anything this time :sweat_smile: